contracts/cw721-ics/src/msg.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
![image](https://user-images.githubusercontent.com/521978/180591677-b633be01-c881-4ee1-93a6-02fda3f8289d.png)

Empty file?

Reply:
lol @Callum-A is a madlad. Removed.